src/lib/with-layout.js

```diff
@@ -84,14 +84,18 @@ export default function withLayout (Component) {
       this.setState({ toggleShowClass: !this.state.toggleShowClass })
     }
 
-    toggleMiniBag() {
-      this.setState({ minibagDisplayed: !this.state.minibagDisplayed })
+    toggleMiniBag(displayed) {
```

Comment:
is there any reason why we need pass displayed through? I'm presuming displayed is a boolean, if so i don't see a need to pass a value through with the action, we can just assume every time the action is called it will change from true/false and do this in the reducer.